lukimgather/settings.py

```diff
@@ -113,6 +115,15 @@
     ],
 }
 
+GRAPHQL_JWT = {
+    "JWT_AUTH_HEADER_PREFIX": "Bearer",
+    "JWT_VERIFY_EXPIRATION": True,
+    "JWT_LONG_RUNNING_REFRESH_TOKEN": True,
+    "JWT_REFRESH_EXPIRATION_DELTA": timedelta(weeks=1),
+    "JWT_SECRET_KEY": os.environ["DJANGO_SECRET_KEY"],
```

Comment:
since SECRET_KEY is already initialized at top. Use that key isntead of reading from env?